packages/web-components/fast-components/src/horizontal-scroll/fixtures/horizontal-scroll.html

```diff
@@ -0,0 +1,257 @@
+    <fast-card style="width: 200px; height: 150px;">Card number 4</fast-card>
+    <fast-card>Card number 5</fast-card>
+    <fast-card style="width: 100px; height: 240px;">Card number 6</fast-card>
+    <fast-card>Card number 7</fast-card>
+    <fast-card style="width: 140px; height: 160px;">Card number 8</fast-card>
+    <fast-card>Card number 9</fast-card>
+    <fast-card style="width: 80px; height: 80px;">Card number 10</fast-card>
+    <fast-card style="width: 150px; height: 150px;">Card number 11</fast-card>
+    <fast-card style="width: 110px; height: 260px;">Card number 12</fast-card>
+    <fast-card>Card number 13</fast-card>
+    <fast-card style="width: 100px; height: 300px;">Card number 14</fast-card>
+    <fast-card style="width: 100px; height: 300px;">Card number 15</fast-card>
+    <fast-card>Card number 16</fast-card>
+</fast-horizontal-scroll>
+
+<h2>Bottom aligned</h2>
+<fast-horizontal-scroll class="right-gradient bottom-align">
+    <fast-flipper slot="previous-flipper" direction="previous"></fast-flipper>
+    <fast-flipper slot="next-flipper"></fast-flipper>
+    <fast-card>Card number 1</fast-card>
+    <fast-card style="height: 75px;">Card number 2</fast-card>
+    <fast-card style="width: 100px; height: 300px;">Card number 3</fast-card>
+    <fast-card style="width: 200px; height: 150px;">Card number 4</fast-card>
+    <fast-card>Card number 5</fast-card>
+    <fast-card style="width: 100px; height: 240px;">Card number 6</fast-card>
+    <fast-card>Card number 7</fast-card>
+    <fast-card style="width: 140px; height: 160px;">Card number 8</fast-card>
+    <fast-card>Card number 9</fast-card>
+    <fast-card style="width: 80px; height: 80px;">Card number 10</fast-card>
+    <fast-card style="width: 150px; height: 150px;">Card number 11</fast-card>
+    <fast-card style="width: 110px; height: 260px;">Card number 12</fast-card>
+    <fast-card>Card number 13</fast-card>
+    <fast-card style="width: 100px; height: 300px;">Card number 14</fast-card>
+    <fast-card style="width: 100px; height: 300px;">Card number 15</fast-card>
+    <fast-card>Card number 16</fast-card>
+</fast-horizontal-scroll>
+
+<h2>Mobile horizontal-scroll</h2>
+<fast-horizontal-scroll class="right-gradient" view="mobile">
+    <fast-card>Card number 1</fast-card>
+    <fast-card>Card number 2</fast-card>
+    <fast-card>Card number 3</fast-card>
+    <fast-card>Card number 4</fast-card>
+    <fast-card>Card number 5</fast-card>
+    <fast-card>Card number 6</fast-card>
+    <fast-card>Card number 7</fast-card>
+    <fast-card>Card number 8</fast-card>
+    <fast-card>Card number 9</fast-card>
+    <fast-card>Card number 10</fast-card>
+    <fast-card>Card number 11</fast-card>
+    <fast-card>Card number 12</fast-card>
+    <fast-card>Card number 13</fast-card>
+    <fast-card>Card number 14</fast-card>
+    <fast-card>Card number 15</fast-card>
+    <fast-card>Card number 16</fast-card>
+</fast-horizontal-scroll>
+
```

Comment:
You may want to add an example where the cards have tab able content? I'd suggest a simple `<button>` in the first card and another one in the last card to see how that effects things.

In experimentation, I added a `<button>` to the first and last card of the default example.
Tabbing to the button on the last card seems to mess up the scroll behavior and gets things out of sync. The component may need to listen for focus events or something to get synced back up.

Reply:
I just tried to repro this by adding a button to the last card - all I notice is that it pulls the card into view and focusses on it. Nothing else is focusable, so shift tab isn't going to go back to anything (there's nothing to focus on but the storybook controls. With everything focusable, this brings them into focus but doesn't "smoothly" scroll, which I think is ok. @SethDonohue what is your expected behavior here?

Reply:
Did you try to scroll with the flippers after tabbing to that button?

Reply:
Sure, that is more clear. No, I did not try and do that. We really shouldn't have to listen for events here - that seems like it would prove costly. @robarbms take a look at Seth's note - anything we can do to ensure that this doesn't get too out of whack?

Reply:
I can see how that might be a problem. I'm using an internal position property to track so that I don't have to touch the DOM every time. I'll explore checking current position each time a scroll is called. It's going to require a fair amount of refactoring as I think I'll need to calculate each "page" position on load rather than just individual item positions.

Reply:
I've redone the code so that it uses scrollLeft rather than translate. This allows you to tab through items in the horizontal-scroll. The scripts now work off what ever scroll position they are currently at.

Reply:
Looks good!